CSC690App/FinalProject
# Updated Setup class

Pushed to feature-branch:

Updated Setup class; Created Article class for corresponding Scene; Created Setup, CellView, and CellData classes.

My Next Todo:
- Add API and parse in MainList and Setup classes.
- Configure the look of the Article class/Scene
- Finalize the look of the Startup class/Scene

To Test:
Just choose "Run" in Xcode.

@abezepeda : Can you take a quick look?  The first screen should just have placeholders for the shown UI elements.  The button on this screen should take you to the second screen (which is just blank right now while I integrate the use of an API into this screen's corresponding MainList class). 
Thanks!  :)
